packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardInputWarning.js

```diff
@@ -1,21 +1,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
 import { Icon } from '../../../../index';
 
 const LoginCardInputWarning = ({ children, className, ...props }) =>
   children && (
-    <div style={{ marginTop: '5px' }} className={className} {...props}>
+    <div {...props} className={classNames('login-pf-input-warning', className)}>
       <Icon type="pf" name="warning-triangle-o" />
       {`   ${children}`}
```

Comment:
optional but not necessary since it's already there...

unsure if we've got styling around using the spaces, but the spacing around children could probably be updated to take into account the normal spacing that happens between elements (basically one space as opposed to multiple), possibly even update the syntax a little
```
<Icon type="pf" name="warning-triangle-o" />
{` `}
{children}
```